share/nisar/defaults/static.yaml

```
@@ -0,0 +1,439 @@
+      # Specification (JPL D-107727).
+      output_hdf5_file:
+      # [OPTIONAL] Directory where the software may write temporary data.
+      # If the directory does not exist, it will be created.
+      # Defaults to a directory named 'scratch' in the current working directory.
+      scratch_dir: scratch
+      # [OPTIONAL] Whether to delete the scratch directory when finished.
+      # Defaults to true.
+      delete_scratch_dir: true
+
+    primary_executable:
+      # [OPTIONAL] Product version string.
+      # Used to populate the '/science/{L,S}SAR/identification/productVersion' dataset
+      # in the output product.
+      # Defaults to '0.0.0'.
+      product_version: "0.0.0"
+      # [OPTIONAL] Digital Object Identifier (DOI) for the product.
+      # Used to populate the '/science/{L,S}SAR/identification/productDoi' dataset in
+      # the output product.
+      # Defaults to '(NOT SPECIFIED)'.
+      product_doi: (NOT SPECIFIED)
+      # [OPTIONAL] Unique 6-digit alphanumeric product version identifier in the science
+      # data production system.
+      # Used to form the granule ID and populate the
+      # '/science/{L,S}SAR/identification/compositeReleaseId' dataset in the output
+      # product.
+      # Must conform to the Composite Release Identifier (CRID) format specified in the
+      # L-SAR Product File Naming Conventions document (JPL D-102255).
+      # Defaults to 'D99999'.
+      composite_release_id: D99999
+      # [OPTIONAL] Processing type.
+      # Used to populate the '/science/{L,S}SAR/identification/processingType' dataset
+      # in the output product.
+      # Must be one of:
+      #   - 'PR': nominal (production)
+      #   - 'OD': custom (off-nominal on-demand processing)
+      # Defaults to 'OD'.
+      processing_type: OD
+      # [OPTIONAL] Mission identifier.
+      # Used to form the granule ID and populate the
+      # '/science/{L,S}SAR/identification/missionId' dataset in the output product.
+      # Defaults to 'NISAR'.
+      mission_id: NISAR
+      # [OPTIONAL] Platform name.
+      # Used to populate the '/science/{L,S}SAR/identification/platformName' dataset
+      # in the output product.
+      # Defaults to 'NISAR'.
+      platform_name: NISAR
+      # [OPTIONAL] Instrument name.
+      # Used to populate the '/science/{L,S}SAR/identification/instrumentName' dataset
+      # in the output product.
+      # Defaults to 'L-SAR'.
+      instrument_name: L-SAR
+      # [OPTIONAL] Radio frequency band.
+      # Used to form the granule ID and populate the
+      # '/science/{L,S}SAR/identification/radarBand' dataset in the output product.
+      # Defaults to 'L'.
+      radar_band: L
+      # [OPTIONAL] Data processing center.
+      # Used to form the granule ID and populate the
+      # '/science/{L,S}SAR/identification/processingCenter' dataset in the output
+      # product.
+      # Defaults to '(NOT SPECIFIED)'.
```

Per https://github.com/isce-framework/isce3/pull/144#discussion_r2399942917

```suggestion
      # '/science/{L,S}SAR/identification/processingCenter' dataset in the output
      # If 'JPL' is specified, the processing center character in the granule ID
      # will be populated with a 'J'; otherwise it will be populated with an 'X'.
      # Defaults to '(NOT SPECIFIED)'.
```